minigalaxy/ui/gametile.py

```diff
@@ -349,7 +432,7 @@ def reload_state(self):
             self.update_to_state(self.state.INSTALLABLE)
         else:
             self.update_to_state(self.state.DOWNLOADABLE)
-        check_upd_thread = threading.Thread(target=self.__check_for_update())
+        check_upd_thread = threading.Thread(target=self.__check_for_update_dlc())
```

Comment:
Not really related to this PR, but renaming this thread to check_update_thread would be better. Feel free to ignore this one, though.